src/neo4j_graphrag/llm/anthropic_llm.py

```diff
@@ -49,62 +58,74 @@ def __init__(
         self,
         model_name: str,
         model_params: Optional[dict[str, Any]] = None,
+        system_instruction: Optional[str] = None,
         **kwargs: Any,
     ):
-        try:
-            import anthropic
-        except ImportError:
+        if anthropic is None:
             raise ImportError(
                 """Could not import Anthropic Python client.
                 Please install it with `pip install "neo4j-graphrag[anthropic]"`."""
             )
-        super().__init__(model_name, model_params)
+        super().__init__(model_name, model_params, system_instruction)
         self.anthropic = anthropic
         self.client = anthropic.Anthropic(**kwargs)
         self.async_client = anthropic.AsyncAnthropic(**kwargs)
 
-    def invoke(self, input: str) -> LLMResponse:
+    def get_messages(
+        self, input: str, chat_history: Optional[list[Any]] = None
+    ) -> Iterable[MessageParam]:
+        messages = []
+        if chat_history:
+            try:
+                MessageList(messages=chat_history)
+            except ValidationError as e:
+                raise LLMGenerationError(e.errors()) from e
+            messages.extend(chat_history)
+        messages.append(UserMessage(content=input).model_dump())
+        return messages
+
+    def invoke(
+        self, input: str, chat_history: Optional[list[Any]] = None
```

Comment:
Isn't `Any` here just a `dict[str, str]`?

Reply:
Yes, I had that initially, but mypy was complaining on it for some reason.

Reply:
IMHO, we can keep `Any` at the moment, maybe just add a comment so that we remember why and we know where to start from if we really want to remove it in the future.